src/ripple/app/paths/Flow.h

```diff
@@ -58,6 +60,7 @@ flow (PaymentSandbox& view,
     bool defaultPaths,
     bool partialPayment,
     bool ownerPaysTransferFee,
+    bool offerCrossing,
     boost::optional<Quality> const& limitQuality,
     boost::optional<STAmount> const& sendMax,
```

Comment:
Here's a look at an approach using an OR-able `enum`:
https://github.com/scottschurr/rippled/commit/8e6e0c9ce3d62276f2415a2582707fc85e37dd1a.
If this looks like an improvement to you I'll cherry pick it into this pull request.